examples/noise-js/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+    <title>Waku noise</title>
+    <link rel="apple-touch-icon" href="./favicon.png" />
+    <link rel="manifest" href="./manifest.json" />
+    <link rel="icon" href="./favicon.ico" />
+  </head>
+  <body>
+    <p>Press F12 to open the console</p>
+    <p><b>Waku Status:</b> <span id="waku-status">connecting...</span></p>
+    <p id="handshake-span">
+      <b>Handshake Status:</b> <span id="handshake-status">-</span>
+    </p>
+    <canvas id="qr-canvas"></canvas>
+    <a href="#" id="qr-url" style="display: none" target="_blank"
+      >Open QR code link in new window instead of scanning it.</a
+    >
+
+    <div id="chat-area" style="display: none">
+      <label for="nick-input">Your nickname</label>
+      <input id="nick-input" placeholder="Choose a nickname" type="text" />
+      <label for="text-input">Message text</label>
+      <input id="text-input" placeholder="Type your message here" type="text" />
+      <button id="send-btn" type="button" disabled>
+        Send message using Light Push
```

Comment:
```suggestion
        Send
```

Protocol used to send message does not matter.